pkg/rancher-desktop/main/commandServer/httpCommandServer.ts

```diff
@@ -80,6 +81,14 @@ export class HttpCommandServer {
         '/v1/extensions/uninstall': [1, this.uninstallExtension],
       },
     } as const,
+    {
+      get:  { '/v1/snapshots': [0, this.listSnapshots] },
+      post: {
+        '/v1/snapshot':         [0, this.createSnapshot],
```

Comment:
By conventional REST protocol, `this.createSnapshot` should be done with `POST /v1/snapshots` (because it adds another entry to the `snapshots` list)